src/clj/rems/db/test_data.clj

```diff
@@ -193,7 +193,14 @@
                   :fi "Tekstialue pituusrajalla"
                   :sv "Textområdet med begränsat längd"}
     :field/optional true
-    :field/max-length 100}])
+    :field/max-length 100}
+
+   {:field/type :phonenumber
+    :field/title {:en "Phone number"
+                  :fi "Puhelinnumero"
+                  :sv "Puhelinnumero"}
+    :field/optional true
+    :field/max-length 20}])
```

Comment:
if we have format checking for phone numbers, do we really need to also support max-length?

Reply:
I do not see the harm in it

Reply:
maximum length for a phone number is 15 digits, and then the plus sign (which actually is just a shorthand and gets replaced with 00 in the phone exchange).

Reply:
I just don't see a valid usecase for supporting max-length, and I think having max-length enabled for this field makes the form editor more complex.

Reply:
I can fix that, should not be an issue